roles/openshift_master/tasks/main.yml

```diff
@@ -104,3 +104,14 @@
   with_items:
   - root
   - "{{ ansible_ssh_user }}"
+
+- name: Update the permissions on the OpenShift client config(s)
+  file:
+    path: "~{{ item }}/.config/openshift/.config"
```

Comment:
is the config file really ".config" ?  That seems pretty horrible to me, or that it may be a bug in this playbook.

 Not blocking on this, just wondering. Let me know and I'll merge if this is actually correct. The rest looks good.